src/Uno.UI/ts/WindowManager.ts

```diff
@@ -733,6 +744,37 @@ namespace Uno.UI {
 			return true;
 		}
 
+		public arrangeElementNativeFast(
+			htmlId: number,
+			top: number,
+			left: number,
+			width: number,
+			height: number,
+			clip: boolean,
+			clipTop: number,
+			clipLeft: number,
+			clipBottom: number,
+			clipRight: number) {
+
+			const element = this.getView(htmlId);
+
+			const style = element.style;
+
+			style.position = "absolute";
+			style.top = top + "px";
+			style.left = left + "px";
+			style.width = width === NaN ? "auto" : width + "px";
+			style.height = height === NaN ? "auto" : height + "px";
+
+			if (clip) {
+				style.clip = `rect(${clipTop}px, ${clipRight}px, ${clipBottom}px, ${clipLeft}px)`;
+			} else {
+				style.clip = "";
+			}
+
+			this.setAsArranged(element);
```

Comment:
It's probably best to avoid duplicating the code above, it's tricky enough :)

Reply:
Fair :)